packages/api/src/auth/jwtStrategy/jwt.module.ts

```diff
@@ -0,0 +1,21 @@
+import { Module } from '@nestjs/common';
+import { ConfigModule, ConfigService } from '@nestjs/config';
+import { JwtModule } from '@nestjs/jwt';
+
+import { JwtStrategy } from './jwt.strategy';
+
+@Module({
+  imports: [
+    JwtModule.registerAsync({
```

Comment:
Use values from the `env.ts`. This way it would be more type-safe and you wouldn't need to register this module asynchronously.